src/SD.cpp

```diff
@@ -354,6 +354,12 @@ boolean SDClass::begin(uint32_t clock, uint8_t csPin) {
          root.openRoot(volume);
 }
 
+//call this when a card is removed. It will allow you to inster and initialise a new card.
+void SDClass::end()
+{
+  root.close();
```

Comment:
Since `SD.begin()` calls `SDCARD_SPI.begin();`.

I'm wondering if a call to `SDCARD_SPI.end();` should be added as well.

Reply:
Not sure. I'll let others decide.

Reply:
Nobody bothered. Can we just keep it like this?